schematic/store/synapse.py

```diff
@@ -79,8 +79,23 @@ def __init__(
 
         self.syn = self.login(token, access_token, input_token)
         self.project_scope = project_scope
+
+
+        # check if "master_fileview" has been set
+        try: 
+            self.storageFileview = CONFIG["synapse"]["master_fileview"]
+        except: 
```

Comment:
can we specify that we're catching a `KeyError` for the try except blocks?

Reply:
Yes. I could fix that.